tests/functional/.gitignore

```diff
@@ -0,0 +1,5 @@
+ubuntu-key/
+fetch/
+vagrant_ssh_config
+disk-*
+*.retry
```

Comment:
I think this is the cleanest way because we don't have any other need for an ``ansible.cfg`` in the scenario directories and I don't want to make it seem like one is required.

Most of the time these will be ran by CI anyway where it's less of a concern.